### ISSUE TITLE
Make it compatible with Laravel 9

### ISSUE THREAD

**Issue by MarcelGDC** [JsonMapper/JsonMapper#143]
Currently this doesn't work with Laravel 9.

When installing through composer you get
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - json-mapper/json-mapper[2.3.0, ..., 2.10.0] require psr/log ^1.1 -> found psr/log[1.1.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - json-mapper/json-mapper[2.11.0, ..., 2.14.0] require psr/simple-cache ^1.0 -> found psr/simple-cache[1.0.0, 1.0.1] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - json-mapper/laravel-package 2.3.0 requires json-mapper/json-mapper ^2.3 -> satisfiable by json-mapper/json-mapper[2.3.0, ..., 2.14.0].
    - Root composer.json requires json-mapper/laravel-package ^2.3 -> satisfiable by json-mapper/laravel-package[2.3.0].
```

I tried to fork the [JsonMapper](https://github.com/JsonMapper/JsonMapper) package and changed the composer.json file to include the simple-cache version 3:
```
"psr/simple-cache": "^1.0 || ^2.0 || ^3.0",
```
And the composer worked. However, when I try to map any model I receive this error
```
PHP Fatal error:  Declaration of JsonMapper\Cache\ArrayCache::get($key, $default = null) must be compatible with Psr\SimpleCache\CacheInterface::get(string $key, mixed $default = null): mixed in /var/www/html/vendor/json-mapper/json-mapper/src/Cache/ArrayCache.php on line 12
```

Is there any way to set up the this plugin work with the Laravel 9?

**Comment by DannyvdSluijs** [comment]
Moved the issue for now to the core package as the issue originates here.

**Comment by DannyvdSluijs** [comment]
Working on addressing this issue in https://github.com/JsonMapper/JsonMapper/pull/156

**Comment by DannyvdSluijs** [comment]
Just released [v2.16.0](https://github.com/JsonMapper/JsonMapper/releases/tag/2.16.0) to address this issue
